nncf/api/compression.py

```diff
@@ -0,0 +1,316 @@
+#
+#  Copyright (c) 2019-2020 Intel Corporation
```

Comment:
We have to update years in copyrights. I don't mind doing that in a different PR.

Reply:
Done